tensorflow/python/framework/config.py

```diff
@@ -653,3 +654,25 @@ def disable_mlir_bridge():
 def disable_mlir_graph_optimization():
   """Disables experimental MLIR-Based TensorFlow Compiler Optimizations."""
   context.context().enable_mlir_graph_optimization = False
+
+
+@tf_export('config.get_cuda_version_used_to_compile_tf')
+def get_cuda_version_used_to_compile_tf():
```

Comment:
Why not a config.get_build_info that returns a dictionary with string keys representing all relevant build info (including cuda and cudnn versions)?

Reply:
Agreed. A dictionary would be great so we do not need to add `get_rocm_version_used_to_compile_tf` or `get_hccl_version_used_to_compile_tf` etc. in the future.

Reply:
Agreed, I've addressed this in my latest commit.